WalletWasabi.Gui/Controls/BusyIndicator.cs

```diff
@@ -0,0 +1,34 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Presenters;
+using Avalonia.Controls.Primitives;
+using Avalonia.Styling;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WalletWasabi.Gui.Controls
+{
+	public class BusyIndicator : ContentControl, IStyleable
```

Comment:
This should inherit TemplatedControl, not ContentControl AFAIKT

Reply:
I used GroupBox as reference implementation:

```
public class GroupBox : ContentControl, IStyleable
	{
```

Reply:
Shall I change this anyway? The question is why did u use ContentControl there?

Reply:
Please do, the groupbox is a control that contains other Content, and has a `Content` property.

Where as your BusyIndicator doesn't.

Reply:
I got errors if I change it. I think it is because I am using it as ContentControl. With BusyIndicator you can wrap your control and if it Loading it will hide the current content. Look at the following xaml: 

https://github.com/zkSNACKs/WalletWasabi/pull/1398/files#diff-be7890d9fbbb4497229be3ff58ae1f32R25

Reply:
Sorry that was my missinterpretation on how you were using it.